source/Lessons/Components/06-Properties.mint

```diff
@@ -60,13 +60,13 @@ module Lessons {
         * Properties can be referenced by name within the component (in styles,
           functions, computed properties, etc...).
 
-        The type definition or default value can be omitted, but not both:
+        The type definition or the default value can be omitted, but not both:
 
         ```mint
         // Type inferred from the default value
         property name = "World"
 
-        // The property is required
```

Comment:
I probably didn't make this clear. When the type definition is provided and no default value it means that when a using the component that property is required and it will result in a type error when omitted.

Reply:
Ah, yes, I misread that. How about this commit?

Reply:
It's perfect, thank you! :pray: